src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/CannotAddOrRemoveVirtualModifier.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+
+#nullable enable
```

Comment:
Thanks for that @smasher164. I wanted to recommend to you using the compiler's nullability feature anyway. Great that you already did it yourself. We might want to also look at the other existing rules and see if we can easily turn nullability on for them as well. Especially as I found some issues when I turned it on in the other Compatibility components.